application/src/main/resources/org/opentripplanner/apis/gtfs/schema.graphqls

```diff
@@ -97,7 +97,7 @@ union CallStopLocation = Location | LocationGroup | Stop
 union RentalPlace = RentalVehicle | VehicleRentalStation
 
 "A feature for a step"
-union StepFeature = Entrance
+union StepFeature = Entrance | EscalatorUse | StairsUse
```

Comment:
The names are inconsistent, and not proper english. `EscalatorUse` means  "using an escalator," "escalator safety," or "escalator etiquette", which is not the same as "usage information for an escalator".

Consider just `Escalator`, `EscalatorFeature` (I dont like this), or `Escalator(Usage)?(Info|Description)?`. 

I also find it starange that there is only one feature for a step (not a colection):
```
"Information about a feature associated with a step e.g. a station entrance or exit."
feature: StepFeature
```

Reply:
I vote for `EscalatorDetails`.